src/validation/coinbase.cairo

```diff
@@ -30,7 +30,12 @@ pub fn validate_coinbase(
     let block_reward = compute_block_reward(block_height);
     assert(total_output_amount <= total_fees + block_reward, 'total output > block rwd + fees');
 
-    // TODO: validate BIP-34 sig script
+    // validate BIP-34 sig script
```

Comment:
Block height check seems to be missing.

Reply:
> Block height check seems to be missing.

The Block height check is there in the `if` statement

Reply:
I mean check if script starts with the current block height

Reply:
> I mean check if script starts with the current block height

@maciejka  How do I extract the block height from the script sig

Reply:
You can read it here: https://learnmeabitcoin.com/technical/mining/coinbase-transaction/